### PR TITLE
Bug 1143206 - [Private Browsing] Sheets displaying incorrect icon/text

### PR DESCRIPTION
**Pull request by KevinGrandon**
https://bugzilla.mozilla.org/show_bug.cgi?id=1143206

### CODE REVIEW

**Comment by try-server-hook** [comment]
[KevinGrandon Kevin Grandon (KevinGrandon)](https://github.com/KevinGrandon) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=861e925dce2a)

File: apps/system/test/unit/app_window_test.js [hunk 1]
```diff
@@ -1960,6 +1960,17 @@ suite('system/AppWindow', function() {
       );
     });
 
+    test('Titilechange event > private browser', function() {
```
Comment (etiennesegonzac) [inline apps/system/test/unit/app_window_test.js]: nit: titlechange

Comment (etiennesegonzac) [inline apps/system/test/unit/app_window_test.js]: I'm pretty sure there's a french pun to make on `titilechange` too :)

Comment (KevinGrandon) [inline apps/system/test/unit/app_window_test.js]: Funny - I just copy/pasted this from another test suite, that's what I get :)

Corrected in both places.

**Comment by try-server-hook** [comment]
[KevinGrandon Kevin Grandon (KevinGrandon)](https://github.com/KevinGrandon) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=0bc4eed2f8e3)

**Pull request merged**